engines/dfc_provider/app/services/enterprise_builder.rb

```diff
@@ -23,6 +23,9 @@ def self.enterprise(enterprise) # rubocop:disable Metrics/AbcSize
       socialMedias: SocialMediaBuilder.social_medias(enterprise),
       websites: [enterprise.website].compact,
     ).tap do |e|
+      # The model strips the protocol and we need to add it:
+      e.websites = e.websites.map { |url| "https://#{url}" }
```

Comment:
Could this be done above on line 24?

Reply:
Yes, but I would need to add a new method because of the conditionals. The website may be blank and we can't just add "https://" to it in that case. Or do you mean adding the `map` on that line? Yes, it would probably fit.

Reply:
Done.

Reply:
I'm not sure if the website could be an empty string, but it might be worth using `compact_blank` to cater for that.

Reply:
Good point. I'll update that.